core/modules/widgets/navigator.js

```diff
@@ -209,6 +209,10 @@ NavigatorWidget.prototype.handleCloseOtherTiddlersEvent = function(event) {
 
 // Place a tiddler in edit mode
 NavigatorWidget.prototype.handleEditTiddlerEvent = function(event) {
+	var editTiddler = $tw.hooks.invokeHook("th-editing-tiddler",event);
```

Comment:
Hi @danielo515 we never use scoped references for $tw because it is our global state, and thus a singleton. The hooks are global, which is why they are referenced via $tw.